src/app/app-routing.module.ts

```diff
@@ -13,7 +13,10 @@ const routes: Routes = [
     path: '',
     loadChildren: () => import('./modules/landing/landing.module').then(m => m.LandingModule)
   },
-
+  {
```

Comment:
Comment this out on prod if it's a staging-only page

Reply:
@Skt2021 I just realized this will not work. I'd suggest rendering/redirecting to 404 on prod instead of commenting it out

Reply:
Done, Added a PageNotFound component